fe1-web/src/features/social/components/ChirpCard.tsx

```diff
@@ -255,16 +245,23 @@ const ChirpCard = ({ chirp, isFirstItem, isLastItem }: IPropTypes) => {
                   {heart}
                 </Text>
               </View>
-              {actionSheetOptions.length > 0 && (
+              {isSender && (
                 <View style={styles.reactionView}>
                   <PoPIconButton
-                    name="options"
-                    testID="chirp_action_options"
-                    onPress={() => showActionSheet(actionSheetOptions)}
+                    name="delete"
+                    testID="delete"
+                    onPress={() => setShowDeleteConfirmation(true)}
                     size="small"
                     buttonStyle="secondary"
                     toolbar
                   />
+                  <ConfirmModal
+                    onConfirmPress={deleteChirp}
+                    visibility={showDeleteConfirmation}
+                    description="Are you sure you want to delete this chirp?"
+                    title="Confirm Deletion"
```

Comment:
We try to put all strings in `src/resources/strings.ts`. While this solution is not perfect it allows for easy translation of the app and without having to search the whole application for static strings :)